src/main/webapp/forms/noarray.dateRO.jsp

```diff
@@ -21,6 +21,70 @@
         Map row = AbstractBase.getMappedRow("SELECT " + zielattribute + " FROM "
                 + zielTabelle + " WHERE ID ='" + id + "'");
 
+        if (row.get("VonTag") == null) {
```

Comment:
Statt alle aktuell existierenden Felder hardzucoden wäre es vermutlich einfacher die null-Werte einmal zentral zu prüfen, siehe #263 Zeilen 32/33 und 37/38. Das Handling für die Spezialwerte null, 0 und -1 ist ja weiter unten schon in diesem Template enthalten und wird korrekt berücksichtigt.